packages/optimizer/lib/transformers/SeparateKeyframes.js

```diff
@@ -125,7 +145,11 @@ class SeparateKeyframes {
     });
 
     // if no rules moved nothing to do
-    if (!keyframesTree.stylesheet.rules.length) return;
+    if (!keyframesTree.stylesheet.rules.length) {
```

Comment:
nit: `array.length === 0` is more readable than `!array.length`

Reply:
done